app/presenters/dlme_thumbnail_presenter.rb

```diff
@@ -17,6 +17,10 @@ def thumbnail_value(image_options)
     super image_options.reverse_merge(loading: 'lazy')
   end
 
+  def default_thumbnail_value(image_options)
+    super(image_options.except(:skip_pipeline))
+  end
+
```

Comment:
will this get magically used somewhere without being called? just wondering if that's a blacklight thing

Reply:
yes, this just overrides the upstream implementation.